index.html

```diff
@@ -398,6 +321,5 @@ <h1>Contact Me</h1>
       </form>
```

Comment:
- [ ] Please I highly recommend sticking to the required [Figma Desktop Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=XWnQyP1APhJu8dzl-0). 😃

_**REQUIRED FIGMA DESIGN**_

![Screenshot (275)](https://user-images.githubusercontent.com/78104544/236277050-b5880814-fe16-46c2-9c67-0318f3f743ee.png)


_**YOUR DESIGN**_

- [ ] Please I highly recommend making changes to the `width` of the `form` to fit the required Figma design. 😄 
- [ ] Please I highly recommend making changes to the `button` with the text "Get in Touch" with `display: flex` and `align-items: center` properties to properly position it. 😄 
- [ ] [OPTIONAL] Please I highly suggest adding hover effects to fit the required Figma design. This is a screenshot of the hover effect design provided in the Figma template. 😄 
![Screenshot (278)](https://user-images.githubusercontent.com/78104544/236279262-adab3344-5cc4-4370-8f0f-32d137606acb.png)

-----------------

![Screenshot (276)](https://user-images.githubusercontent.com/78104544/236277392-0d9649dd-6935-4133-b0df-0840d4e7c21a.png)